### PR TITLE
Possible edits for adding UseSSL for MySQL

### PR DESCRIPTION
**Pull request by EHCanadian**
Small edits to hopefully enable support for java's MySQL UseSSL.

Uses true/false boolean

Default false. Having this set stops the console from throwing out WARN about UseSSL

(Not a java dev. Seemed to compile fine on travis-ci.org). Unable to test as I do not have a java build env.

### CODE REVIEW

**Comment by EHCanadian** [comment]
Was trying to get travis to upload builds to test. Made a mess with the push. Original 3 files have edits.
Anything else related to travis.yml can be ignored.

**Comment by games647** [comment]
Why did you closed it? We could revert the changes on the travis.yml and then merge all commits together. It's not a big thing, I can do that for you if you want to.

File: src/main/java/com/github/games647/lagmonitor/LagMonitor.java [hunk 1]
```diff
@@ -132,11 +132,12 @@ private void setupMonitoringDatabase() {
             String host = getConfig().getString("host");
             int port = getConfig().getInt("port");
             String database = getConfig().getString("database");
+            Boolean usessl = getConfig().getBoolean("usessl");
```
[the remark this answers is not in the document]
Comment (EHCanadian) [reply src/main/java/com/github/games647/lagmonitor/LagMonitor.java]: Noted. Just trying to follow the existing coding pattern used. :)

File: src/main/java/com/github/games647/lagmonitor/storage/Storage.java [hunk 2]
```diff
@@ -30,7 +30,7 @@
     private final Logger logger;
     private final String prefix;
 
-    public Storage(Logger logger, String host, int port, String database, String user, String pass, String prefix) {
+    public Storage(Logger logger, String host, int port, String database, Boolean usessl, String user, String pass, String prefix) {
```
[the remark this answers is not in the document]
Comment (EHCanadian) [reply src/main/java/com/github/games647/lagmonitor/storage/Storage.java]: Noted.

**Comment by EHCanadian** [comment]
> Why did you closed it? We could revert the changes on the travis.yml and then merge all commits together. It's not a big thing, I can do that for you if you want to.

I don't need credits.. 1st time trying to use this git to offer help. Made a bit of a mess.. As soon as you add the edits I will pull the fork on my end and we will never speak of this again lol..

**Comment by games647** [comment]
>I don't need credits.. 1st time trying to use this git to offer help.

Yeah, the webinterface often isn't very helpful. Using it locally integrated into the development environment like Eclipse, IntelliJ, ... is usually the best way, but requires some knowledge.

>As soon as you add the edits I will pull the fork on my end and we will never speak of this again lol..

To be honest, I'm thankful of every pull request event the smallest one like typos or this one. I only checked if this project compiles and then used one command to squash the commits and merge it. I don't even had to boot up my heavy IDE since you did all necessary changes.

**Comment by EHCanadian** [comment]
Well. She works. I tested the #150 & #151 build over at ci.codemc.org
[](https://ci.codemc.org/job/Games647/job/LagMonitor/150/)

**Pull request closed without merging**